site/package.json

```diff
@@ -11,10 +11,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "@geoarrow/geoarrow-wasm": "^0.2.0-beta.4",
+    "@mui/icons-material": "^5.15.19",
+    "@mui/material": "^5.15.18",
```

Comment:
Any idea of the impact on this change on our bundle size? I hesitate to introduce Material as a dependency unless we are going to use a lot of their components.

We currently are using Infima for styling and components, but looks like a popover is missing. I personally am a fan of https://www.radix-ui.com/primitives/docs/components/popover because you can selectively import the components you use (e.g. https://github.com/placemark/placemark/blob/main/package.json#L58).

Reply:
I could definitely transition to this. I've used Material in the past, so it is what I know, but if Radix provides more 'customizable' importing I could switch to that.